deploy_windows_files.ps1

```diff
@@ -49,14 +49,20 @@ if ($clean_deployment_folder) {
     Invoke-RemoteCommand -Command $clean -Arguments $deployment_folder_path
 }
 
-[Byte[]]$zip = Get-Content -Path $source_zip_file_path -Encoding Byte
-$zip_size = (Get-Item -Path $source_zip_file_path).Length / 1KB
+Write-Output "Copy file: $source_zip_file_path"
+
+$copy_session = New-PSSession $server -SessionOption $so -UseSSL -Credential $credential
```

Comment:
If we're setting up this session, is there a reason to not use it for the other remote calls?